sdk/cosmosdb/cosmos/src/client/ChangeFeed/ChangeFeedForEpkRange.ts

```diff
@@ -393,11 +405,22 @@ export class ChangeFeedForEpkRange<T> implements ChangeFeedPullModelIterator<T>
         type: Constants.HttpHeaders.IfNoneMatch,
         condition: feedRange.continuationToken,
       };
+    } else if (this.startFromNow) {
+      feedOptions.initialHeaders[Constants.HttpHeaders.IfNoneMatch] = "*";
     }
 
     if (this.startTime) {
       feedOptions.initialHeaders[Constants.HttpHeaders.IfModifiedSince] = this.startTime;
     }
+
+    if (
```

Comment:
Just for my clarification, can you explain the significance of the the `useAllVersionsAndDeletesFeed` and `useIncrementalFeed` flags on `feedOptions` when you already have the `changeFeedMode`?

Reply:
`changeFeedMode` is included in this new version of change feed iterator. There is one older change feed iterator (which will be deprecated in future release) which worked just for partition keys and `useIncrementalFeed` is getting used there. Removing `useIncrementalFeed` will be a breaking change. 
Internally both the iterators use same code to build the request and `changeFeedMode` gets mapped to either `useIncrementalFeed` or `useAllVersionsDeletesFeed`. I have updated the code now to hide `useAllVersionsDeletesFeed` from public documentation so it doesn't get used directly.